### PR TITLE
ILS: Handle invalid quantities

### PR DESCRIPTION
**Pull request by kkrampa**
@gcapalbo 
We need to provide a transation for that message in transifex:
Here is the translation: Tafadhali rudia kwa kuingiza Namba kwenye idadi: <number>

### CODE REVIEW

File: custom/ilsgateway/tanzania/handlers/randr.py [hunk 1]
```diff
@@ -38,6 +38,14 @@ def _handle(self, help=False):
                 self.respond(SUBMITTED_REMINDER_DISTRICT)
             else:
                 quantities = [self.args[1], self.args[3], self.args[5]]
+
+                for quantity in quantities:
+                    try:
+                        int(quantity)
```
Comment (gcapalbo) [inline custom/ilsgateway/tanzania/handlers/randr.py]: are all quantities integers and none of them ever have decimal parts?

Comment (kkrampa) [inline custom/ilsgateway/tanzania/handlers/randr.py]: Moved behavior from v1: https://github.com/dimagi/logistics/blob/tz-master/logistics_project/apps/tanzania/handlers/randr.py#L78

File: custom/ilsgateway/tanzania/reminders/__init__.py [hunk 2]
```diff
@@ -73,6 +73,9 @@
     "Reply with 'submitted A <number of R&Rs submitted for group A> "
     "B <number of R&Rs submitted for group B>'"
 )
+SUBMITTED_INVALID_QUANTITY = _(
+    'Not a valid number: <number>'
```
Comment (gcapalbo) [inline custom/ilsgateway/tanzania/reminders/__init__.py]: sorry, just realized this - this should be `%(number)s`

**Comment by kkrampa** [comment]
@gcapalbo Updated but it seems build needs to be restarted

**Comment by gcapalbo** [comment]
restarted build, good to merge on pass

**Comment by gcapalbo** [comment]
(updated translation)

**Pull request merged**